Packs/AMP/Integrations/CiscoAMPEventCollector/README.md

```diff
@@ -0,0 +1,45 @@
+This is the Cisco AMP event collector integration for XSIAM.
+This integration was integrated and tested with version v1 of CiscoAMPEventCollector.
```

Comment:
What is v1? the intention is to mention the CISCO AMP version not the XSOAR collector
```suggestion
This integration was integrated and tested with version v1 of CiscoAMPEventCollector.
```

Reply:
v1 is the Cisco API version